app/Console/Commands/Paie/PollSalaryPaymentsCommand.php

```
@@ -0,0 +1,40 @@
+        $runs = SalaryPaymentRun::query()
+            ->whereNotNull('bridge_payment_request_id')
+            ->whereIn('status', [
+                SalaryPaymentStatus::AWAITING_VALIDATION->value,
+                SalaryPaymentStatus::PROCESSING->value,
+            ])
+            ->get();
+
+        foreach ($runs as $run) {
+            try {
+                $service->pollRun($run);
+                $this->line("Run {$run->id} -> {$run->fresh()->status->value}");
```

PHPStan (2.2.7)

[error] Cannot call method fresh() on stdClass.


(method.nonObject)